tests/test_mdlp.py

```diff
@@ -184,6 +184,18 @@ def test_mdlp_iris():
         [ 1.,  0.,  2.,  2.],
         [ 1.,  1.,  2.,  2.],
         [ 1.,  0.,  2.,  2.]]
-
     assert_array_equal(transformed, expected,
                        err_msg="MDLP output is inconsistent with previous runs.")
+
+
+def test_cutpoints():
+    # mdlp cutpoints produced by Spark mdlp implementation by Sergio Ramírez #
+    feature_set = {'x1': [4.1, 19.9, 180.76, 2398.13], 'x2': [-2192.84, -549.56, 2.93, 14.25, 22.09, 336.18, 175492.79], 'x3': [508, 1205, 1681, 2938, 3196], 'x4': [1997, 4057, 7006, 11716, 19665, 1063223]}
+    df = pd.read_csv("test.csv")
```

Comment:
Henry,

I uploaded a new data set in the test folder.

From: Henry Lin <notifications@github.com<mailto:notifications@github.com>>
Reply-To: hlin117/mdlp-discretization <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, 18 August 2016 at 09:10
To: hlin117/mdlp-discretization <mdlp-discretization@noreply.github.com<mailto:mdlp-discretization@noreply.github.com>>
Cc: Jaco DuToit <jaco.dutoit@jumo.world<mailto:jaco.dutoit@jumo.world>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [hlin117/mdlp-discretization] add cutpoint test from Spark MDLP implementation (#2)


In tests/test_mdlp.py<https://github.com/hlin117/mdlp-discretization/pull/2#discussion_r75257689>:

>      assert_array_equal(transformed, expected,
>                         err_msg="MDLP output is inconsistent with previous runs.")
> +
> +
> +def test_cutpoints():
> +    # mdlp cutpoints produced by Spark mdlp implementation by Sergio Ramírez #
> +    feature_set = {'x1': [4.1, 19.9, 180.76, 2398.13], 'x2': [-2192.84, -549.56, 2.93, 14.25, 22.09, 336.18, 175492.79], 'x3': [508, 1205, 1681, 2938, 3196], 'x4': [1997, 4057, 7006, 11716, 19665, 1063223]}
> +    df = pd.read_csv("test.csv")


Can you verify that the dataset I attached is this test.csv?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/hlin117/mdlp-discretization/pull/2/files/72aac7e5880d82446842626895fb4e5df49813fb#r75257689>, or mute the thread<https://github.com/notifications/unsubscribe-auth/APZnNynsZnxJjcE8J-R-Vle6apZ8WSOrks5qhAVXgaJpZM4JnMZN>.